apps/ledger-live-mobile/src/components/ServicesWidget/Protect/useProtect.ts

```diff
@@ -0,0 +1,59 @@
+import { useFeature } from "@ledgerhq/live-common/featureFlags/index";
+import {
+  useLearnMoreURI,
+  useAlreadySubscribedURI,
+} from "@ledgerhq/live-common/hooks/recoverFeatureFlag";
+import { useCallback, useMemo } from "react";
+import { Linking } from "react-native";
+import { track } from "../../../analytics";
+import { urls } from "../../../config/urls";
+
+export function useProtect() {
+  const servicesConfig = useFeature("protectServicesMobile");
+
+  const learnMoreURI = useLearnMoreURI(servicesConfig);
+  const alreadySubscribedURI = useAlreadySubscribedURI(servicesConfig);
+
+  const onLearnMore = useCallback(() => {
+    const url = `${learnMoreURI}&source=${urls.recoverSources.myLedger}`;
+    Linking.canOpenURL(url).then(() => Linking.openURL(url));
+  }, [learnMoreURI]);
+
+  const onAlreadySubscribe = useCallback(() => {
+    const url = `${alreadySubscribedURI}&source=${urls.recoverSources.myLedger}`;
+    Linking.canOpenURL(url).then(() => Linking.openURL(url));
+  }, [alreadySubscribedURI]);
```

Comment:
We don't need useCallback here as we expose functions onClickCard / onClickSignIn / onClickFreeTrial from the hook. So it will not protect from rerendering.